core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2052,6 +2052,8 @@ class Log(@volatile private var _dir: File,
             )
           }
           // FIXME: this code path involves not only data plane segments but also KRaft metadata logs.  Should find a way to distinguish after moving to KRaft.
+          
+          // XXX: An internal dashboard depends on parsing this warn log line. Get SRE reviews before changing the format.
```

Comment:
Why is there an "XXX:" here?

Reply:
Just like FIXME/TODO, it's sort of a common way to shout out things in comments
https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

In some text editor themes, XXX even renders red 
![Screenshot 2023-01-31 at 13 05 55](https://user-images.githubusercontent.com/935681/215883082-0e50584e-edcd-4f43-8f35-0c4353e541e6.png)

Reply:
Thanks!